ClassicAssist/UO/Data/Enums.cs

```diff
@@ -157,4 +157,13 @@ public enum CharacterListFlags
         NewMovementSystem = 0x4000,
         NewFeluccaAreas = 0x8000
     }
+
+    [Flags]
```

Comment:
Remove the [Flags], you can't specify multiple hues...

https://docs.microsoft.com/en-us/dotnet/api/system.flagsattribute?view=net-6.0

Reply:
Ahh ok. I will remove. Ty so much ...